automation/src/test/scala/org/broadinstitute/dsde/test/api/MethodLaunchSpec.scala

```diff
@@ -0,0 +1,117 @@
+package org.broadinstitute.dsde.test.api
+
+import java.util.UUID
+
+import akka.actor.ActorSystem
+import akka.testkit.TestKit
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.UserPool
+import org.broadinstitute.dsde.workbench.fixture.{BillingFixtures, MethodData, WorkspaceFixtures}
+import org.broadinstitute.dsde.workbench.service.{AclEntry, Rawls, RestException, WorkspaceAccessLevel}
+import org.broadinstitute.dsde.workbench.service.test.{CleanUp, RandomUtil}
+import org.broadinstitute.dsde.workbench.util.Retry
+import org.scalatest.{FreeSpecLike, Matchers}
+import org.broadinstitute.dsde.workbench.fixture._
+import spray.json._
+import DefaultJsonProtocol._
+import org.scalatest.time.{Minutes, Seconds, Span}
+import org.scalatest.concurrent.Eventually
+
+
+class MethodLaunchSpec extends TestKit(ActorSystem("MySpec")) with FreeSpecLike with Matchers
+  with CleanUp with RandomUtil with Retry with Eventually
+  with BillingFixtures with WorkspaceFixtures with MethodFixtures {
+
+  val methodConfigName: String = SimpleMethodConfig.configName + "_" + UUID.randomUUID().toString
+  val operations = Array(Map("op" -> "AddUpdateAttribute", "attributeName" -> "participant1", "addUpdateAttribute" -> "testparticipant"))
+  val entity: Array[Map[String, Any]] = Array(Map("name" -> "participant1", "entityType" -> "participant", "operations" -> operations))
+
+  "launch workflow with input not defined" in {
```

Comment:
It'd be nice for the test name to make it clear that launching a workflow with undefined input will throw an exception